### PR TITLE
St svetlana seda lesson2 level2

### PR DESCRIPTION
**Pull request by java0711**
Please check my homework for level 2. TY

### CODE REVIEW

File: src/main/java/students/student_svetlana_seda/lesson_2/level_2/HelloLana.java [hunk 1]
```diff
@@ -0,0 +1,16 @@
+package students.student_svetlana_seda.lesson_2.level_2;
+
+import java.util.Scanner;
+
+public class HelloLana {
+    public static void main(String[] args) {
+
+            String n;
```
Comment (antdolg1) [inline src/main/java/students/student_svetlana_seda/lesson_2/level_2/HelloLana.java:8]: Bad variable name, please fix.

Also format lines.

**Review by antdolg1 — CHANGES_REQUESTED** [review]
Fix naming.

File: src/main/java/students/student_svetlana_seda/lesson_2/level_2/HelloLana.java (hunk 1 above)
Comment (antdolg1) [reply src/main/java/students/student_svetlana_seda/lesson_2/level_2/HelloLana.java]: Use auto format: `Ctrl+Alt+L`

**Review by antdolg1 — CHANGES_REQUESTED** [review]
Apply autoformat on your code in class HelloLana.java

File: src/main/java/students/student_svetlana_seda/lesson_2/level_2/HelloLana.java (hunk 1 above)
Comment (java0711) [reply src/main/java/students/student_svetlana_seda/lesson_2/level_2/HelloLana.java]: Thanks, corrected now.

**Comment by java0711** [comment]
All corrected. Ty.

**Comment by java0711** [comment]
Please confirm what is wrong? What changes need to be done? Autoformat done and the name fixed... ty

**Review by antdolg1 — APPROVED** [review]

**Review by antdolg1 — APPROVED** [review]

**Pull request merged**